src/translations/en.json

```diff
@@ -220,7 +220,7 @@
 
     "descLfVoltageInitMode": "Voltage initialisation mode: ",
     "descLfTransformerVoltageControlOn": "Activate on-load tap changers: ",
-    "descLfNoGeneratorReactiveLimits": "Take the reactive limits of generators into account: ",
+    "descLfUseReactiveLimits": "Take the reactive limits of generators into account: ",
```

Comment:
Translations have to be modified ? UseReactiveLimits does not apply to generators only anymore ?